commands/proposal_submission.go

```diff
@@ -369,6 +369,18 @@ func checkNewMarketChanges(change *protoTypes.ProposalTerms_NewMarket) Errors {
 		errs.AddForProperty("proposal_submission.terms.change.new_market.changes.position_decimal_places", ErrMustBeWithinRange7)
 	}
 
+	lppr, err := num.DecimalFromString(changes.LpPriceRange)
+	if err != nil {
+		errs.AddForProperty("proposal_submission.terms.change.new_market.changes.lp_price_range", ErrIsNotValidNumber)
+	} else {
+		if lppr.IsNegative() || lppr.IsZero() {
```

Comment:
could be
```suggestion
		if lppr.Cmp(num.DecimalZero()) <= 0 {
```

not specifically better I suppose though

Reply:
I think mine is more readable for ppl not that familiar with the library